File: src/main/java/com/ddd/trainings/proposal/infra/TrainingProposalRepo.java
package com.ddd.trainings.proposal.infra;

import com.ddd.trainings.proposal.domain.ProposalId;
import com.ddd.trainings.proposal.domain.TrainingProposal;
import com.ddd.utils.BaseRepo;

public interface TrainingProposalRepo
    extends BaseRepo<TrainingProposal, ProposalId> {

}

File: src/Ddd/Trainings/Proposal/Adapters/TrainingProposalRepo.cs
using Ddd.Trainings.Proposal.Domain;
using Ddd.Trainings.Proposal.Domain.Ports;
using Ddd.Utils;

namespace Ddd.Trainings.Proposal.Infra;

public class TrainingProposalRepo : BaseRepo<TrainingProposal, ProposalId>, ITrainingProposalRepo
{
  public override List<TrainingProposal> FindAll()
  {
    return base.FindAll().ToList();
  }
} 